.github/ISSUE_TEMPLATE/feature-rfc-request.yml

```diff
@@ -0,0 +1,116 @@
+name: 🚀 RFC Release highlight for Feature proposed
+description: Submit an RFC for a new PyTorch feature
+labels: ["feature-rfc-request"]
+# todo: Stable and Unstable as labels
+
+body:
+- type: markdown
+  attributes:
+    value: >
+      #### Note: Please write your RFC request in English to ensure it can be understood and addressed by the development team.
+- type: textarea
+  attributes:
+    label: Release highlight for Feature proposed
+    description: >
+      Example: “A torch.special module, analogous to SciPy's special module.”
+- type: input
+  id: contact
+  attributes:
+    label: Point(s) of contact
+    description: How can we get in touch with you if we need more info?
+    placeholder: ex. github username
+  validations:
+    required: false
+- type: dropdown
+  attributes:
+    label: Release Mode (pytorch/pytorch features only)
+    description: |
+      If "out-of-tree", please include the GH repo name
+    options:
+      - In-tree
+      - Out-of-tree
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Out-Of-Tree Repo
+    description: >
+      please include the GH repo name
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Description and value to the user
+    description: >
+      Please provide a brief description of the feature and how it will benefit the user.
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Link to design doc, GitHub issues, past submissions, etc
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: What feedback adopters have provided
+    description: >
+      Please list users/teams that have tried the feature and provided feedback. If that feedback motivated material changes (API, doc, etc..), a quick overview of the changes and the status (planned, in progress, implemented) would be helpful as well.
+  validations:
+    required: false
+- type: dropdown
+  attributes:
+    label: Plan for documentations / tutorials
+    description: |
+      Select One of the following options
+    options:
+      - Tutorial exists
+      - Will submit a PR to pytorch/tutorials
+      - Will submit a PR to out-of-tree repo
```

Comment:
This implies we don't plan on hosting tutorials for features implemented out of tree?

Reply:
Hi @albanD and @Xia-Weiwen refactored to say :
```
- Tutorial exists
- Will submit a PR to pytorch/tutorials
- Will submit a PR to a repo
- Tutorial is not needed
 ``` 
Not to mix with out-of-tree feature.